mgmt/rest/server.go

```diff
@@ -80,6 +80,40 @@ type Config struct {
 	RestAuthPassword string `json:"rest_auth_password,omitempty"yaml:"rest_auth_password,omitempty"`
 }
 
+const (
+	CONFIG_CONSTRAINTS = `
+			"restapi" : {
+				"type": ["object", "null"],
+				"properties" : {
+					"enable": {
+						"type": "boolean"
+					},
+					"https" : {
+						"type": "boolean"
+					},
+					"rest_auth": {
+						"type": "boolean"
+					},
+					"rest_auth_password": {
+						"type": "string"
+					},
+					"rest_certificate": {
+						"type": "string"
+					},
+					"rest_key" : {
+						"type": "string"
+					},
+					"port" : {
+						"type": "integer",
+						"minimum": 1,
+						"maximum": 65535
```

Comment:
Idk but that is the current behavior. I think it's actually a bug in the current state b/c if you give it 0 it will select a port for you but then not output what port was selected.